modules/ossm-uguide-routing-ingress.adoc

```diff
@@ -0,0 +1,228 @@
+
+    $ gcloud compute firewall-rules create allow-gateway-http --allow tcp:$INGRESS_PORT 
+
+    $ gcloud compute firewall-rules create allow-gateway-https --allow tcp:$SECURE_INGRESS_PORT
++
+*  _Minikube:_
+
+    $ export INGRESS_HOST=$(minikube ip)
++
+*  _Docker For Desktop:_
+
+    $ export INGRESS_HOST=127.0.0.1
+
++
+*  _Other environments, such as IBM Cloud Private_:
+
+    $ export INGRESS_HOST=$(oc get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
+
+[id="ossm-ingress-config-_{context}"]
+== Configuring ingress using an {ProductShortName} gateway
+
+An ingress gateway describes a load balancer operating at the edge of the mesh that receives incoming HTTP/TCP connections.
+
+It configures exposed ports and protocols, but does not include any traffic routing configuration. Traffic routing for ingress traffic is configured with {ProductShortName} routing rules, which is the same way internal service requests are configured.
+
+Configure a gateway on port 80 for HTTP traffic.
+
+1.  Create an {ProductShortName} `Gateway`:
+
+    $ oc apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: httpbin-gateway
+    spec:
+      selector:
+        istio: ingressgateway # use {ProductShortName} default gateway implementation
+      servers:
+      - port:
+          number: 80
+          name: http
+          protocol: HTTP
+        hosts:
+        - "httpbin.<<yourdomain>>.com"
+    EOF
++
+1.  Configure routes for traffic entering via the gateway:
+
+    $ oc apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: httpbin
+    spec:
+      hosts:
+      - "httpbin.<<yourdomain>>.com"
+      gateways:
+      - httpbin-gateway
+      http:
+      - match:
+        - uri:
+            prefix: /status
+        - uri:
+            prefix: /delay
+        route:
+        - destination:
+            port:
+              number: 8000
+            host: httpbin
+    EOF
++
+Those commands create a virtual service configuration for the `httpbin` service containing two route rules that allow traffic for paths `/status` and `/delay`.
++
+The gateways list specifies that only requests through your `httpbin-gateway` are allowed. All other external requests are rejected with a 404 response.
++    
+Internal requests from other services in the mesh are not subject to these rules but instead will default to round-robin routing. To apply these rules to internal calls as well, you can add the special value `mesh` to the list of `gateways`. Since the internal hostname for the service is probabaly different, for example `httpbin.default.svc.cluster.local`, from the external one, you must also add it to the `hosts` list.
++
+1.  Access the `httpbin` service with cURL:
```

Comment:
You seem to be instructing the user to run and verify a command but this won't work because there were no previous instructions to deploy this httpbin application. This tutorial only tells the user to deploy the bookinfo one.

Reply:
That's a good point. I'll work on that.

Reply:
I decided to remove that example rather than add the instructions for installing the httpbin sample. I think this pr is big enough as is. I can add the httpbin sample or other items in a later release.